docs/man/msync.md

```diff
@@ -0,0 +1,146 @@
+muntar 1 "May 2023" Manta "Manta Commands"
```

Comment:
Looks like there are still a few references to muntar in this file.

Reply:
Thanks. Fixed.